<ui version="4.0" >
 <class>WaitFormClass</class>
 <widget class="QWidget" name="WaitFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lTimeWait" >
     <property name="text" >
      <string>Время ожидания:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="bExitQueue" >
     <property name="text" >
      <string>Выйти из очереди</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lCntWait" >
     <property name="text" >
      <string>Кол-во ожидающих</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="lFight" >
     <property name="text" >
      <string>В бою</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="lCntCommon" >
     <property name="text" >
      <string>Общее кол-во</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
